.github/workflows/semantic-pull-requests.yml

```diff
@@ -0,0 +1,21 @@
+name: "Validate PRs"
```

Comment:
Just change this up to `Validate PR name` or similar and we’re good to go 🎉

Reply:
Done 🚀